keys-addresses.asciidoc

```diff
@@ -352,7 +352,7 @@ How can you tell if the software library you are using is FIPS-202 SHA-3 or Kecc
 An easy way to tell is to use a _test vector_, an expected output for a given input. The test most commonly used for a hash function is the _empty input_. If you run the hash function with an empty string as input you should see the following results:
 
 [[sha3_test_vectors]]
-.Testing whether the SHA3 library you are using is Keccak-256 of FIP-202 SHA-3
+.Testing whether the SHA3 library you are using is Keccak-256 or FIP-202 SHA-3
```

Comment:
Please, follow this link for [FIPS PUB 202](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf)